### PR TITLE
Lower limit and pagination to fix property definitions from freezing 

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

*Please describe.*  
*If this affects the frontend, include screenshots.*  

Fix for #5376 

Screen Shot 2021-08-19 at 10 42 29 AM

I think a couple culprits for ^ is that we're trying to load too many values at once in the backend and the frontend, so I've lowered the pagination limit on the API call and also the table because I doubt someone needs to see all 9999 property definitions at once?

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

**Comment by liyiy** [comment]
It's kind of hard to test this properly without the 10k property definitions we have on cloud 😅

File: frontend/src/models/propertyDefinitionsModel.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const propertyDefinitionsModel = kea<
             {
                 loadPropertyDefinitions: async (initial?: boolean) => {
                     const url = initial
-                        ? 'api/projects/@current/property_definitions/?limit=5000'
+                        ? 'api/projects/@current/property_definitions/?limit=500'
```
Comment (macobo) [inline frontend/src/models/propertyDefinitionsModel.ts:28]: Hmm. Is this change needed. Not sure what the implications here are.


Originally w e added this since netdata has a bunch of custom properties. and loading them all took forwever. However with new properties filter in insights not sure if we need this :thinking:

Comment (liyiy) [reply frontend/src/models/propertyDefinitionsModel.ts]: Hmm yeah you have a point here, if this works for the filters it should be fine for the property definitions page too. I undid this change, let's see if just paginating the frontend for it will take care of the freezing. (Though I did encounter the API call itself crashing when I went to the property definitions page)

**Review by macobo — APPROVED** [review]
